### DISCUSSION TITLE
New point release perhaps?

### DISCUSSION THREAD

**Discussion by leonjza** [containrrr/watchtower#1080] [Releases]
Hey! First of all, thanks for the amazing work ya'll are putting into this project. It's incredibly useful.

I was wondering when the next release was planned? We're eagerly waiting for #1055 to land with some of the fixes included there.

Thanks!

**Comment by simonaronsson** [comment]
Hi,

Last time I spoke with @piksel about whether we should do a new release we agreed there were still a couple of things we'd like to have added prior to that. But I would say that we, hopefully, aren't too far away.

Best,
Simme

**Reply by thespad** [reply]
+1 on this, we could really do with #990 being in a released build to help with organising our stats.

**Comment by thespad** [comment]
Don't mean to be a pain here but there have been nearly 75 PRs merged since the last release, some of them with major changes or bug fixes, none of which most users are benefitting from.

**Reply by simonaronsson** [reply]
I get that, and I am very sympathetic to the pain it causes. Currently, it's just a matter of none of us having enough bandwidth to get a release out the door. It is still high on my todo-list but will unfortunately require some more time.
